src/ui-kit/preloader/preloader.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Icon } from 'antd'
+
+import './preloader.scss'
+
+const Preloader = () => {
+  return <Icon type="loading" className="ui-kit__loading" />
```

Comment:
use the explicit return:
```
const Preloader => <Icon ..../>
```

Reply:
and by the way, there's a Loader component I think in the **antd** already, could you double-check that please ?